packages/ui-view/src/View/index.tsx

```diff
@@ -100,7 +100,7 @@ type Props = {
   withVisualDebug?: boolean
   makeStyles?: (...args: any[]) => any
   styles?: any
-  dir?: any // TODO: PropTypes.oneOf(Object.values(bidirectional.DIRECTION)),
```

Comment:
This change is there because I wanted to test the feature of the PR, so because this is the only file which has examples in it's package, only the `ui-view` package's stories will be pushed to `Chromatic`.